docs/src/man/basics.md

```diff
@@ -0,0 +1,2042 @@
+# First Steps with DataFrames.jl
+
+The first and important part to work with DataFrames.jl, is to install it.
+You can do it using the following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+or,
+
+```julia
+julia> ] # `]` should be pressed
+
+(@v1.6) pkg>  add DataFrames
+```
+
+If you want to make sure everything works as expected you could run the tests bundled
+with DataFrames.jl, but be warned that it will take more than 30 minutes.
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
+
+Additionally, it is recommended to check the version installed with `st` command (this tutorial is for 1.x releases).
+```julia
+julia> ]
+
+(@v1.6) pkg> st DataFrames
+      Status `C:\Users\TeAmp0is0N\.julia\environments\v1.6\Project.toml`
+  [a93c6f00] DataFrames v1.1.1
+```
+
+Throughout the rest of the tutorial we will assume that you have installed the DataFrames.jl package and
+have already typed `using DataFrames` which loads the package:
+```jldoctest dataframe
+julia> using DataFrames
+```
+
+The most fundamental type provided by DataFrames.jl is `DataFrame`, where typically
+each row is interpreted as an observation and each column as a feature.
+
+# Constructors and Basic Utility Functions
+
+## Constructors
+
+In this section you will see many ways to create a `DataFrame` using the constructor.
+First, let's create an empty `DataFrame`:
+```jldoctest dataframe
+julia> df = DataFrame()
+0×0 DataFrame
+```
+
+Or, we could call the constructor using keyword arguments to add columns to the `DataFrame`:
+```jldoctest dataframe
+julia> df = DataFrame(A=1:3, B=5:7, fixed=1)
+3×3 DataFrame
+ Row │ A      B      fixed
+     │ Int64  Int64  Int64
+─────┼─────────────────────
+   1 │     1      5      1
+   2 │     2      6      1
+   3 │     3      7      1
+```
+note that in column `:fixed` scalars gets automatically broadcasted.
+
+To move forward with the tutorial you need to install the CSV.jl package in your environment.
+In order to do so run the following commands:
+```julia
+julia> using Pkg
+julia> Pkg.add("CSV")
+```
+Make sure you have CSV.jl in a version that is at least 1.0.
+
+Now, we will explore how to load a CSV file into a `DataFrame`. Unlike Python's Pandas `read_csv`
+the you need two packages to accomplish this: CSV.jl and DataFrames.jl. As the first
+step, you have to load the libraries you will use. In our case CSV.jl and DataFrames.jl. In order to
+read the file in we will use the `CSV.read` function.
+```jldoctest dataframe
+julia> using CSV
+
+julia> german_ref = CSV.read(joinpath(dirname(pathof(DataFrames)),
+                                      "..", "docs", "src", "assets", "german.csv"),
+                              DataFrame)
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
```

Comment:
TODO: check column eltypes after CSV.jl release.

Reply:
So, it's for me right?

Reply:
TODO are comments that should not be resolved now. I leave them to make sure that what is left in the comment is verified before merging of the PR as we are not sure how long it will take to merge it.